src/pid/status.rs

```diff
@@ -192,6 +192,9 @@ named!(parse_status_state<State>,
           | tag!("Z (zombie)") => { |_| State::Zombie }));
 
 named!(parse_command<String>,   delimited!(tag!("Name:\t"),      parse_line,         line_ending));
+#[cfg(any(all(target_os = "android", target_arch = "arm"), target_os = "freebsd", target_os = "macos"))]
```

Comment:
You can remove "freebsd" and "macos", because we don't need to support platforms other than Linux.

Reply:
True, but as things stand, one cannot develop on MacOS using this crate then, even if the code will only run on Linux.